extras/rya.streams/api/src/test/java/org/apache/rya/streams/api/queries/InMemoryQueryRepositoryTest.java

```diff
@@ -83,7 +82,7 @@ public void initializedWithPopulatedChangeLog() throws Exception {
         final QueryChangeLog changeLog = new InMemoryQueryChangeLog();
         final QueryRepository queries = new InMemoryQueryRepository( changeLog, SCHEDULE );
         try {
-            queries.startAndWait();
+            queries.startAsync();
```

Comment:
Why?  Did you have a source article for these changes?

Reply:
The old method was removed by Guava, it was previously deprecated: https://guava.dev/releases/16.0/api/docs/com/google/common/util/concurrent/Service.html#startAndWait()

Reply:
I didn't bother with awaitRunning() etc for test code because it didn't seem to need it, but I included it for all production code.